### ISSUE TITLE
Allow SteamPlay to choose proton, or system wine prefix

### ISSUE THREAD

**Issue by Gibbz** [ValveSoftware/steam-for-linux#5674]
Id love to see support not only for proton, but also to allow the user to pick their custom wine prefix/install from the steamplay settings.

This would be very handy to be able to set per game also!

For example in my wine prefix i have some special settings to get certain games working that dont work otherwise.

**Comment by ronlaws86** [comment]
+1 for this. Also i'm sifting through the structure and it is unclear where if anywhere proton gets its settings and saves such for games. At the moment it's impossible if not just undocumented to run things like Nexus Mod Manager for a game for example.

**Comment by juliotux** [comment]
Some games run perfectly in my default wine, but not run in Proton. Choose between Proton and my default wine should be great.

In addition, the choice of wineprefix may fix some issues, like all data being erased when game is uninstalled, same things (like Uplay) being installed in multiple wineprefixes, while they can be installed just once.

**Comment by FlyingWombat** [comment]
First, Thanks Valve for making this awesome feature and supporting wine!

Just thought I'd throw in my 2 cents:  
Since Valve won't want to support every wine version out there (nor should they have to), I think it'd be more beneficial for everyone, to make an interface for external applications to manage wine versions, prefixes, and attributes.  
This way, Valve would only have to officially support their own wine builds. While users could enjoy maximum compatibility with the Steam Linux client, by continuing to use the wine management tools that they're used to (e.g. Lutris, PlayOnLinux, etc.) 

TL;DR
It'd be great if we could make use of the features of Steam's native Linux client, while using our wine customizations.

**Comment by andrebadaia** [comment]
I would also like it very much!

**Comment by Kreyren** [comment]
+1 would make lots of things way easier

**Comment by trymeouteh** [comment]
I would like the ability to use system wine version and a way to use a custom wine version one has installed on their computer such as _/home/USER/Documents/Wine 4.20-staging/bin/wine_

**Comment by newbthenewbd** [comment]
This really needs to become a feature, and I would like to offer an additional viewpoint as to the _why_.

My Steam install in Wine, at the time of writing, uses up exactly 879.8 megabytes on my drive (not counting the size of the installed games, of course). It works reliably, but is quite tedious to use in the long term - especially when I want to update the games. The `WINEPREFIX` that it runs in, like any other `WINEPREFIX` ~~that you can buy in every `WINEPREFIX` store~~, takes up about one and a half a gigabyte by default. This pushes the total overhead of this setup up to about 2.38 gigabytes.

I have just made an attempt to switch over to Proton. This was the third such attempt of mine in the recent months. And such were, once more, my findings: before I abandoned the experiment, Proton managed to grow in size to 1.8 gigabytes while creating its own prefix. I am not certain if this prefix was only meant to be used for the single lightweight game that I attempted to run, so I will assume the most favorable possibility - that this grown up Proton would service all my games without further bloat and the overhead would peak at 1.8 gigabytes.

_So what's the problem_, I hear you asking, _clearly Proton is lighter?_ Well, I answer - I (and sure not only I) also use Wine to launch a multitude of other applications, thus the one and half a gigabyte stays there anyways. And I may want to run games that fail to run in Proton but work in Wine, as mentioned by the kind fellows above, which would also require maintaining the Wine install of Steam. This pushes the total overhead of the Proton setup up to 1.5+1.8=**3.3** or 1.5+0.88+1.8=**4.18** gigabytes, respectively. This increase is not worth the switch - especially so when it's only going to be a partial switch.

The addition of these two seemingly minor features has the potential to turn that situation upside down. Namely: a new _Wine Standalone_ option in the Proton version selection menu that would simply execute the games using `/usr/bin/wine`, and a method to specify the `WINEPREFIX` (also per game) if a person wishes to use one that is different from the default `~/.wine`.

Suddenly, even if some of the Proton magic were to have to stay (weighing, say, no more than 500 megabytes, and there's [prior art](https://github.com/xomachine/SteamForwarder) that is much lighter) in order to satisfy the DRMs, not only would my and many others' envisioned Steam Play setups be much lighter than the two separate Steam installations for Windows and Linux games that we are currently using, they would also be much more comfortable, increasing the certainty that if a game can be gotten to run on Linux at all, it will run _in this very client_.

So, **TL;DR**: We really need this. Linux gaming really needs this. Please add.

**Comment by frostworx** [comment]
Just found this request via https://www.reddit.com/r/linux_gaming/comments/nekaoc/how_to_use_already_system_installed_wine_on_steam/
My [steamtinkerlaunch](https://github.com/frostworx/steamtinkerlaunch) has exactly that [feature](https://github.com/frostworx/steamtinkerlaunch/wiki/Wine-Support). An own wineprefix is used besides the proton one, so the original one remains untouched.
